packages/backend/src/peripherals/database/ForcedTradeOfferRepository.ts

```diff
@@ -123,17 +124,22 @@ export class ForcedTradeOfferRepository extends BaseRepository {
     super(database, logger)
 
     /* eslint-disable @typescript-eslint/unbound-method */
-
     this.add = this.wrapAdd(this.add)
     this.update = this.wrapUpdate(this.update)
     this.updateTransactionHash = this.wrapUpdate(this.updateTransactionHash)
     this.findById = this.wrapFind(this.findById)
-    this.countInitial = this.wrapAny(this.countInitial)
     this.getInitial = this.wrapGet(this.getInitial)
-    this.countActiveByPositionId = this.wrapAny(this.countActiveByPositionId)
     this.getByPositionId = this.wrapGet(this.getByPositionId)
+    this.getUserOffersByStarkKey = this.wrapGet(this.getUserOffersByStarkKey)
```

Comment:
The name `getUserOffersByStarkKey` is a bit unclear here. I guess `getByStarkKey` is not good here because we compare against maker *and taker*? If so, maybe this should be named `getByMakerOrTakerStarkKey`?